src/test/java/com/orzechazo/eshop/services/OrderServiceImplIT.java

```diff
@@ -0,0 +1,118 @@
+package com.orzechazo.eshop.services;
+
+import com.orzechazo.eshop.bootstrap.tests.BootstrapUsersAndOrders;
+import com.orzechazo.eshop.domain.Order;
+import com.orzechazo.eshop.domain.dto.OrderDto;
+import com.orzechazo.eshop.exceptions.BadRequestException;
+import com.orzechazo.eshop.exceptions.ResourceNotFoundException;
+import com.orzechazo.eshop.repositories.OrderRepository;
+import com.orzechazo.eshop.repositories.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(SpringExtension.class)
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class OrderServiceImplIT {
+
+    @Autowired
+    private OrderRepository orderRepository;
+    @Autowired
+    private UserRepository userRepository;
+    private OrderServiceImpl orderService;
+    private UserServiceImpl userService;
+    private static final LocalDateTime DATE_TIME = LocalDateTime.of(2024,1,1,0,0);
+    private static final String DB_USER_LOGIN = BootstrapUsersAndOrders.DB_USER_LOGIN;
+    private static final String DB_ORDER_ID1 = BootstrapUsersAndOrders.DB_ORDER_ID1;
+    private int DB_DEFAULT_ORDER_COUNT;
+    private BootstrapUsersAndOrders bootstrapUsersAndOrders;
+    @BeforeEach
+    void setUp() {
+        bootstrapUsersAndOrders = new BootstrapUsersAndOrders(orderRepository,userRepository);
+        bootstrapUsersAndOrders.loadData();
+
+        userService = new UserServiceImpl(userRepository);
+        orderService = new OrderServiceImpl(orderRepository, userService);
+        DB_DEFAULT_ORDER_COUNT = bootstrapUsersAndOrders.getOrders().size();
+    }
+
+    @Test
+    void getAllOrders() {
+        //given
+        List<String> dbOrderIdList = bootstrapUsersAndOrders.getOrders().stream()
+                .map(Order::getOrderId).toList();
+        //when
+        List<OrderDto> orderDtos = orderService.getAllOrders();
+        List<String> orderDtosIdList = orderDtos.stream()
+                .map(OrderDto::getOrderId).toList();
+        //then
+        assertThat(dbOrderIdList).containsExactlyInAnyOrderElementsOf(orderDtosIdList);
```

Comment:
You are comparing dbOrderIdList and orderDtosIdList and on the first sight it is hard to tell which is expected value and which is actual value returned from service, pls rename it